open_optical_gating/cli/optical_gater_server.py

```diff
@@ -139,20 +152,11 @@ def analyze(self, pixelArray):
         if self.state == "sync":
             # Using previously-determined reference peiod, analyse brightfield frames
             # to determine predicted trigger time for prospective optical gating
-            (trigger_response, current_phase, current_time_s) = self.sync_state(
-                pixelArray
-            )
-
-            # Logs results and processing time
-            time_fin = time.time()
-            self.timestamp.append(current_time_s)
-            self.phase.append(current_phase)
-            self.processing_rate_fps.append(1 / (time_fin - time_init))
             self.predicted_trigger_time_s.append(
                 None
             )  # placeholder - updated inside sync_state
```

Comment:
Is there a reason for doing it this way? I found it a bit odd to create a placeholder here. I guess the reason is historical (seeing the old, deleted code in this diff), but any reason not to move it into self.sync_state() now?